ui/packages/catalog-ui-search/src/main/webapp/js/model/User.spec.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Newb question... why did you need to add this file?

Reply:
Didn't have to, just wanted to add some unit tests for the fix.

Reply:
gotcha, thanks 👍